packages/core/src/icon/utils/icon.classnames.ts

```diff
@@ -33,10 +33,9 @@ export function getUpdateSizeStrategy(size: string) {
 
 export function updateIconSizeStyle(el: CdsIcon, size: string) {
   const updateStrategy = getUpdateSizeStrategy(size);
-  const val = pxToRem(parseInt(size));
-
   switch (updateStrategy) {
     case SizeUpdateStrategies.ValidNumericString:
+      const val = pxToRem(parseInt(size));
```

Comment:
Might need to move `val` into a `let` on line 36 and then assign it to `val` here.